reference-architecture/osp-dns/fragments/install_python.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+#
+# Ensure that Python is installed for Ansible
+#
+echo "Installing Python, DNF and Selinux packages for Ansible"
+[ -f /etc/fedora-release ] && dnf install -y python python-dnf libselinux-python
```

Comment:
They're not needed for rhel.  In fact I think they're not needed for Fedora *if* you run ansible with python3 on Fedora, but I had not seen that done until yesterday.